torch/distributed/tensor/debug/visualize_sharding_example.py

```diff
@@ -0,0 +1,77 @@
+# OMP_NUM_THREADS=1 torchrun --nproc_per_node=4 visualize_sharding_example.py
```

Comment:
why OMP_NUM_THREADS? i think for a visualization example this level of tuning should be unimportant?

Reply:
I included it solely to suppress the following warning.

```
W0423 20:24:18.427000 85647 site-packages/torch/distributed/run.py:792] *****************************************
W0423 20:24:18.427000 85647 site-packages/torch/distributed/run.py:792] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed.
W0423 20:24:18.427000 85647 site-packages/torch/distributed/run.py:792] *****************************************
```